network-area-diagram/src/test/java/com/powsybl/nad/NetworkAreaDiagramTest.java

```diff
@@ -92,21 +92,45 @@ void testVoltageFilteredDiagramTwoBounds() {
     }
 
     @Test
-    void testVoltageFilteredDiagramLowBound() {
+    void testVoltageFilteredDiagramLowerBound() {
         Network network = IeeeCdfNetworkFactory.create14();
         Path svgFileVoltageFilter = fileSystem.getPath("nad-test-voltage-filter.svg");
         NetworkAreaDiagram.draw(network, svgFileVoltageFilter, new NadParameters(), VoltageLevelFilter.createNominalVoltageLowerBoundFilter(network, List.of("VL4"), 120, 2));
         assertEquals(toString("/IEEE_14_bus_voltage_filter1.svg"), getContentFile(svgFileVoltageFilter));
     }
 
     @Test
-    void testVoltageFilteredDiagramHighBound() {
+    void testVoltageFilteredDiagramUpperBound() {
         Network network = IeeeCdfNetworkFactory.create14();
         Path svgFileVoltageFilter = fileSystem.getPath("nad-test-voltage-filter.svg");
         NetworkAreaDiagram.draw(network, svgFileVoltageFilter, new NadParameters(), VoltageLevelFilter.createNominalVoltageUpperBoundFilter(network, List.of("VL4"), 180, 2));
         assertEquals(toString("/IEEE_14_bus_voltage_filter2.svg"), getContentFile(svgFileVoltageFilter));
     }
 
+    @Test
+    void testVoltageFilteredDiagramNoVoltageLevelIdInput() {
+        Network network = IeeeCdfNetworkFactory.create14();
+        Path svgFileVoltageFilter = fileSystem.getPath("nad-test-voltage-filter.svg");
+        NetworkAreaDiagram.draw(network, svgFileVoltageFilter, new NadParameters(), VoltageLevelFilter.createNominalVoltageFilter(network, 120, 180));
+        assertEquals(toString("/IEEE_14_bus_voltage_filter1.svg"), getContentFile(svgFileVoltageFilter));
+    }
+
+    @Test
+    void testVoltageFilteredDiagramLowerBoundNoVoltageLevelIdInput() {
+        Network network = IeeeCdfNetworkFactory.create14();
+        Path svgFileVoltageFilter = fileSystem.getPath("nad-test-voltage-filter.svg");
+        NetworkAreaDiagram.draw(network, svgFileVoltageFilter, new NadParameters(), VoltageLevelFilter.createNominalVoltageLowerBoundFilter(network, 120));
```

Comment:
It would be nice to have a difference in the results of this test and the previous one.
For instance, with `20` as lower bound there is an additional voltage level in the resulting svg.

Reply:
You are right! Done :)

Reply:
Thank you!